src/components/KasulaNavbar.jsx

```diff
@@ -81,7 +81,7 @@ function KasulaNavbar() {
   return (
     <>
       <Navbar expand="lg" className="bg-normal sticky-top">
-        <Container>
+        <Container fluid>
```

Comment:
This makes the container responsive but the dropdown of the right side appears incomplete (only one half of it is visible).

Reply:
![image](https://github.com/UB-ES-2023-A2/kasula-frontend/assets/72074467/cd2fd13d-e3c0-4063-a1f7-fd976ceca5ab)
It works for me, how can I reproduce the problem?

Reply:
![Screenshot 2023-11-21 at 12 44 10](https://github.com/UB-ES-2023-A2/kasula-frontend/assets/47271218/cd200f92-d032-4522-8de9-49906592ed39)

I guess in some way it might depend on the screen, because as you can see in the screenshot, I have the vertical & horizontal scrollers, stating that "there is something else" from the current view.